app/controllers/topics_controller.rb

```diff
@@ -12,6 +13,7 @@ def new
 
   def create
     @topic = Topic.create(topic_params)
```

Comment:
Obviously I missed this in an earlier pull request...

You're creating the topic on this line and then checking that the topic save works, but it is already saved (or the save failed because of validation problems) by the create. 
This line should therefore say:
```
@topic = Topic.new(topic_params)
```